apps/demo-game/src/services/PeriodResultService.ts

```diff
@@ -0,0 +1,73 @@
+import { PayloadPeriodResult, PayloadPeriodResultEnd } from '@gbl-uzh/platform'
+import { debugLog } from '@gbl-uzh/platform/dist/lib/util'
+import { produce } from 'immer'
+import { PlayerRole } from '../settings/Constants'
+import {
+  OutputPeriodResultFactsInit,
+  OutputResultFacts,
+  PeriodResultFactsInit,
+  ResultFacts,
+} from '../types/facts'
```

Comment:
Is it better to import from ../types/index and take the ones from facts AND Period (line below)?